playPrism-web/src/modules/user/components/profile/profile.component.html

```diff
@@ -1,3 +1,4 @@
+<app-header/>
```

Comment:
Please, pass app-header in app.component.html file and import SharedModule to app component

![image](https://github.com/NURE-DigitalDream/PlayPrismWebApp/assets/40624950/bcaec214-4388-4ac0-85ef-4c98c7292566)